widgets/GoogleAdsMultiReportBatchSource-batchsource.json

```diff
@@ -0,0 +1,155 @@
+{
+  "metadata": {
+    "spec-version": "1.5"
+  },
+  "configuration-groups": [
+    {
+      "label": "Basic",
+      "properties": [
+        {
+          "widget-type": "textbox",
+          "label": "Refresh token",
+          "name": "refreshToken"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Client ID",
+          "name": "clientId"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Client Secret",
+          "name": "clientSecret"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Developer token",
+          "name": "developerToken"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Customer ID",
+          "name": "clientCustomerId"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "Start Date",
+          "name": "startDate"
+        },
+        {
+          "widget-type": "textbox",
+          "label": "End Date",
+          "name": "endDate"
+        }
+      ]
+    },
+    {
+      "label": "Advanced",
+      "properties": [
+        {
+          "name": "reportFormat",
+          "label": "Format",
+          "widget-type": "select",
+          "widget-attributes": {
+            "default": "CSV",
+            "values": [
```

Comment:
Values should be human readable not machine readable.

Reply:
What do you mean by human readable
for the format name?
Decoding of an abbreviation or something else?